crates/provider-sdk/src/rpc_client.rs

```diff
@@ -132,7 +139,12 @@ impl RpcClient {
         let subject = make_uuid();
         let issuer = self.key.public_key();
         let target_url = crate::url(&target, Some(&method));
-        let topic = rpc_topic(&target, &self.lattice);
+        let topic = rpc_topic(&target, &self.lattice, &self.wit_resolve);
+
+        // TODO: RPC topic is going to be different and method-dependent here
+        // QUESTION: what is the method here when we're dealing with WIT-native wRPC calls?
+        //   Some variation of the WIT method? `<package>:<namespace>/<interface>/<method>`?
```

Comment:
Yep, you have it right here so far as I understand. So the method will be in the topic like this

Reply:
Yeah I think the answer for this is actually in the code, and I haven't looked it up (since we already handle custom WIT contracts) - so I'll use what's there *for now*, until `wrpc` lib lands and things change possibly.